project-templates/import-records/Readme

```diff
@@ -0,0 +1,122 @@
+Requirements
+
+  node >= 4 https://nodejs.org/
+  make      https://www.gnu.org/software/make/
+  jq        https://stedolan.github.io/jq/
+
+Make does not support filenames with special characters unfortunately.  Remove
+special characters in your filenames.
+
+Make is not included on OSX by default, you need to install XCode or use a
+package manager like [Brew](https://brew.sh/) or
+[Nix](https://nixos.org/nix/).
+
+Import Steps:
+
+  1) Gather source data, export from a database into linked spreadsheets.  By
+     linked I mean related data in two spreadsheets can be joined with a shared
+     column.
+
+  2) Format and convert
+
+    PLACES_LEVEL_3=households.csv \
```

Comment:
Should I as a reader understand what PLACES_LEVEL_3 means? Why not level 2, etc…